internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -236,7 +236,7 @@ func itemToAPI(
 		}
 	}
 
-	if omitValue && isRequired {
+	if omitValue && isRequired || key == "basic_auth_username" || key == "basic_auth_password" {
```

Comment:
@byashimov as I anticipated, it's a one line fix, at least in the current version of the generator